packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -34,26 +34,26 @@ $status-indicator-size: 1em * 0.5;
     height: $status-indicator-size;
```

Comment:
@jeffbelser-okta How do we handle values like this, with no token mapping available? 

Would this be an appropriate case for a `status.json` file that describes Status-only tokens?

Reply:
@edburyenegren-okta we can leave this as is (a sass variable) for now since it isn't part of the acceptance criteria listed in the JIRA or our requirements for end of January MVP.

Lets talk more about what happens next at Office Hours today this is a great topic.